### ISSUE TITLE
Obsoletion: differentiate RNA polymerase activity

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#23779]
The pre-period definitions for 
DNA-directed 5'-3' RNA polymerase activity (GO:0003899)
Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1).

and
RNA polymerase II activity (GO:0001055)
Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1).

are identical

**Comment by pgaudet** [comment]
The same reaction is catalyzed by all RNA polymerases; i think these should be merged and we should distinguish the various polymerases by Protein Complexes. 

Thoughts?

**Comment by ValWood** [comment]
That would work for me.

**Comment by ValWood** [comment]
I have been trying to find out about this for a while
https://github.com/pombase/curation/issues/2033

so what was the decision on process?
Will the specific polymerase only be mentioned in the CC term now?

**Comment by ValWood** [comment]
@pgaudet so should I merge 
GO:0001054 RNA polymerase I activity
GO:0001055 RNA polymerase II activity
GO:0001056 RNA polymerase III activity
up to DNA-directed 5'-3' RNA polymerase activity (GO:0003899)

**Comment by deustp01** [comment]
The three RNA polymerases involve distinct protein families as catalysts and accessory proteins, recognize distinct elements in genomic DNA to initiate RNA synthesis, and of course mediate the synthesis of structurally and functionally distinct RNAs, so merging all of them seems a bit extreme compared to GO's treatment of DNA polymerase activity or aldehyde lyase activity. Just saying ...

**Comment by ValWood** [comment]
I'm ambivalent whether they stay or go. 
We need adequate differentia if they stay.
I'm not sure if different families or accessory factors   is a reason to keep them (this is true for many catalytic activities described by a single term (e.g. Ub ligase ).
I'm also not sure about the binding (this is orthogonal to the activity).
I'll wait for further instructions. No hurry.

**Comment by pgaudet** [comment]
I dont see any reason to keep any of these: 
'RNA polymerase I activity'
'RNA polymerase II activity'
'RNA polymerase III activity'
'RNA polymerase IV activity'
'RNA polymerase V activity'
'single subunit type RNA polymerase activity'

even 'DNA primase activity' really is the same activity. 

The BP should be the distinguishing factor that will group all relevant activities involved in polI, II, III, etc transcription. It would be good to check that the genes annotated to these respective terms have the correct BP (can this be done with the matrix?)

**Comment by ValWood** [comment]
It will tell us if annotations are missing, but difficult to do because it is either all species or individual species.

A better option might be to  identify the PANTHER families and get PAINT  annotation to the relevant  BP/CC terms

**Comment by ValWood** [comment]
I can do that when I do the ticket, and provide @marcfeuermann  with. a list

**Comment by pgaudet** [comment]
Good idea, they should be in the same family - I presume

**Comment by ValWood** [comment]
Yes they are all conserved, but some are in every complex.


Systematic ID | Gene name | Product description
-- | -- | --
SPAC2F7.06c | pol4 | DNA polymerase X family
SPAC6B12.10c | spp1 | DNA primase catalytic subunit Spp1
SPBC17D11.06 | spp2 | DNA primase large subunit Spp2
SPCC1259.03 | rpa12 | DNA-directed RNA polymerase complex I TFIIS subunit Rpa12
SPAC1687.01 | rpc19 | DNA-directed RNA polymerase I and III subunit Rpc19
SPBC1289.07c | rpc40 | DNA-directed RNA polymerase I and III subunit Rpc40
SPBC4C3.05c | nuc1 | DNA-directed RNA polymerase I complex large subunit Nuc1
SPBC1718.03 | ker1 | DNA-directed RNA polymerase I complex subunit Ker1
SPBP23A10.07 | rpa2 | DNA-directed RNA polymerase I complex subunit Rpa2
SPBC11G11.05 | rpa34 | DNA-directed RNA polymerase I complex subunit Rpa34
SPBC3B9.07c | rpa43 | DNA-directed RNA polymerase I complex subunit Rpa43
SPAC23C4.15 | rpb5 | DNA-directed RNA polymerase I, II and III subunit Rpb5
SPCC1020.04c | rpb6 | DNA-directed RNA polymerase I, II and III subunit Rpb6
SPBC14C8.12 | rpb8 | DNA-directed RNA polymerase I, II and III subunit Rpb8
SPBC19C2.03 | rpc10 | DNA-directed RNA polymerase I, II and III subunit Rpc10
SPAC1B3.12c | rpb10 | DNA-directed RNA polymerase I, II, and III subunit Rpb10
SPBC337.14 | rpb4 | DNA-directed RNA polymerase II complex subunit Rpb4
SPAPYUG7.04c | rpb9 | DNA-directed RNA polymerase II complex TFIIS subunit Rpb9
SPBC651.08c | rpc1 | DNA-directed RNA polymerase III complex large subunit Rpc1
SPAC22A12.05 | rpc11 | DNA-directed RNA polymerase III complex subunit Rpc11
SPAPB1E7.10 | rpc17 | DNA-directed RNA polymerase III complex subunit Rpc17
SPAC4G9.08c | rpc2 | DNA-directed RNA polymerase III complex subunit Rpc2
SPBC2G5.07c | rpc25 | DNA-directed RNA polymerase III complex subunit Rpc25
SPBC839.12 | rpc31 | DNA-directed RNA polymerase III complex subunit Rpc31
SPCC290.02 | rpc34 | DNA-directed RNA polymerase III complex subunit Rpc34
SPCC330.13 | rpc37 | DNA-directed RNA polymerase III complex subunit Rpc37
SPCC18.07 | rpc53 | DNA-directed RNA polymerase III complex subunit Rpc53
SPAPB1E7.03 | rpc82 | DNA-directed RNA polymerase III complex subunit Rpc82
SPAC26H5.12 | rpo41 | mitochondrial RNA polymerase Rpo41
SPBC947.13 | rba50 | RNA polymerase II associated protein
SPAC3A12.07 | rpb11 | RNA polymerase II complex subunit Rpb11
SPAC23G3.01 | rpb2 | RNA polymerase II complex subunit Rpb2
SPBC28F2.12 | rpb1 | RNA polymerase II large subunit Rpb1
SPCC1442.10c | rpb3 | RNA polymerase II subunit 3

**Comment by pgaudet** [comment]
Good point!

**Comment by ValWood** [comment]
Eugh, my product lines are inconsistent. I hate that  (note to self- fix the bottom 4)

**Comment by pgaudet** [comment]
Note that the complex terms: 

'RNA polymerase I complex'
'RNA polymerase II, core complex'
'RNA polymerase III complex'
'RNA polymerase IV complex'
'RNA polymerase V complex'

use these MFs in their logical definitions: 
[RNA polymerase I complex](http://purl.obolibrary.org/obo/GO_0005736)
'DNA-directed RNA polymerase complex'
 and ('part of' some nucleolus)
 and ('capable of' some 'RNA polymerase I activity')

The merge will results in equivalent classes; so the 'capable of' statement needs to be moved as asserted superclass.

**Comment by raymond91125** [comment]
To confirm, the decision is to obsolete (replaced with GO:0003899 DNA-directed 5'-3' RNA polymerase activity);

- [ ] GO:0003896 DNA primase activity
- [ ] GO:0001054 RNA polymerase I activity
- [ ] GO:0001055 RNA polymerase II activity
- [ ] GO:0001056 RNA polymerase III activity
- [ ] GO:0001057 RNA polymerase IV activity
- [ ] GO:0001058 RNA polymerase V activity
- [ ] GO:0001064 single subunit type RNA polymerase activity
- [ ] GO:0001065 mitochondrial single subunit type RNA polymerase activity
- [ ] GO:0001066 plastid single subunit type RNA polymerase activity

and obsolete all regulation terms for they should be MF?

- [ ] GO:1903934 positive regulation of DNA primase activity
- [ ] GO:1903932 regulation of DNA primase activity
- [ ] GO:1903623 negative regulation of RNA polymerase III activity
- [ ] GO:1903622 regulation of RNA polymerase III activity

**Comment by ValWood** [comment]
I don't think that DNA primase should be obsoleted as that's a different activity from a polymerase
v

**Comment by raymond91125** [comment]
If we 'merge up' all RNA polymerases but not primase, we are left with a single child for DNA-directed 5'-3' RNA polymerase activity.
That is, if it's OK to also merge up single subunit polymerases.
GO:0001064 single subunit type RNA polymerase activity
...GO:0001065 mitochondrial single subunit type RNA polymerase activity
...GO:0001066 plastid single subunit type RNA polymerase activity

**Comment by ValWood** [comment]
Ignore my comment about primase, I think that is also OK to merge. I was having a 'moment'!

**Comment by krchristie** [comment]
For historical record, we have merged these polymerase activities into one term previously. Then, we ended up reinstating the specific terms when we wanted to link MF terms to the processes they are involved in. If you are dropping MF to BP  links, that concern will not matter, but if you want the links between MF and BP, you do need the more specific terms. 

Please be sure that this merge is what you want as it is harder to re-generate the specificity than to merge.

**Comment by krchristie** [comment]
> I don't think that DNA primase should be obsoleted as that's a different activity from a polymerase v

Considering that DNA primase is a type of RNA polymerase, I'd like you know what your distinction for DNA primase being a different activity than "a polymerase" is.

**Comment by raymond91125** [comment]
> For historical record, we have merged these polymerase activities into one term previously. Then, we ended up reinstating the specific terms when we wanted to link MF terms to the processes they are involved in. If you are dropping MF to BP links, that concern will not matter, but if you want the links between MF and BP, you do need the more specific terms.
> 
> Please be sure that this merge is what you want as it is harder to re-generate the specificity than to merge.

I don't recall this history. But perhaps 1 MF -> many BP is OK.



> > I don't think that DNA primase should be obsoleted as that's a different activity from a polymerase v
> 
> Considering that DNA primase is a type of RNA polymerase, I'd like you know what your distinction for DNA primase being a different activity than "a polymerase" is.

Val has retracted this comment.

**Comment by krchristie** [comment]
> > For historical record, we have merged these polymerase activities into one term previously. Then, we ended up reinstating the specific terms when we wanted to link MF terms to the processes they are involved in. If you are dropping MF to BP links, that concern will not matter, but if you want the links between MF and BP, you do need the more specific terms.
> > Please be sure that this merge is what you want as it is harder to re-generate the specificity than to merge.
> 
> I don't recall this history. 

Perhaps it predates you. I was the person who originally pushed to merge into 1 term, debating the idea with Micheal Ashburner. 

> But perhaps 1 MF -> many BP is OK.

It depends on what the links are used for. If these links are part of what makes a logical definition sufficient, then it may be a problem to condense all these RNA polymerase activities into 1 term.

While it is true that the catalytic enzymatic activity is the same, there are many other interactions of these polymerase activities that make them non-redundant within cells. The current MF terms for the RNA polymerase activities are defined not only with the enzymatic activity but with which type of promoter they utilize.

**Comment by raymond91125** [comment]
@krchristie points taken and thanks for the warning. I don't see the merge conflicts with our current guidelines and practices. Unless @ValWood decides to do otherwise, I'd go ahead with the merge.

**Comment by ValWood** [comment]
I think it's standard GO policy to only have a single tern to represent the same reaction. See @pgaudet comment above "The BP should be the distinguishing factor that will group all relevant activities involved in polI, II, III, etc transcription."
So,  for me obsoletion is correct based on current ontology and annotation practice.

**Comment by vanaukenk** [comment]
@raymond91125 - I was just checking the term replacements as part of the Noctua maintenance outage and was wondering if GO:0001054 RNA polymerase I activity got obsoleted.
There are still annotations to that term in Noctua, but my understanding is that they should have been replaced by the parent.
Thanks.

**Comment by raymond91125** [comment]
Hum, so how it is missed. I'll fix that. Thanks.

**Comment by vanaukenk** [comment]
Thank you @raymond91125 !

**Comment by raymond91125** [comment]
@vanaukenk Thanks for the alert. Checked all the terms are obsoleted.

**Comment by srengel** [comment]
SGD doesn't agree with obsoleting these terms. There are distinct types of RNA polymerases (Pol I, II, and III) that differ in their subunit composition and the specific types of RNA they transcribe, with Pol I making ribosomal RNA, Pol II transcribing messenger RNA, and Pol III producing transfer RNA and small ribosomal RNAs; essentially, each polymerase has a specialized function based on the genes it targets

**Comment by raymond91125** [comment]
Thanks. The argument was that the MFs are essentially the same. The genes may be distinguished by BP annotations. Would you like to discuss this on an Annotation call @srengel?

**Comment by deustp01** [comment]
> The argument was that the MFs are essentially the same.

I'm with @srengel on this one. The distinctions among the RNA polymerases, based on distinct substrate specificities, catalytic features, and regulatory mechanisms are at least as substantial as the ones used to distinguish exo- and endopeptidase activites, and orthogonally metallo- and serine peptidases, even though all of these enzyes mediate the hydrolysis of peptide bonds.

**Comment by ValWood** [comment]
I wanted to keep these for a long time, but @pgaudet convinced me they are describing the same activity.
This is clear from the definition. The peptidases which have different cleavage specificities have different reactions.

This is how I justified it to myself:
- The catalytic part of the polymerase doing the 
nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). is the same for i, II and III.

-The specificity for different regions of DNA is not provided by the catalytic activity, it's provided by non catalytic subunits. (i.e. rpb1 CTD domain  in the case of RNA polymerase II acting as an adaptor to recruit factors to control initiation, elongation, surveillance, splicing, termination and export - but this is an independent  from the catalytic activity.
For example, see the collection of MFs here https://www.pombase.org/gene/SPBC28F2.12
Similarly, the specificity for different genes is not controlled by the catalytic activity but by external regulatory elements, so this should be modelled as part of the broader process rather than the activity.


Based on how we represent other activities it is more consistent to represent  specific polymerase activities and processes in annotation as 
GO:0003899 | DNA-directed 5'-3' RNA polymerase activity 
involved in transcription by RNA polymerase II

In summary, the differences in the exo- and endopeptidase activities are intrinsic to the peptidases. The differences between the polymerase substrates are external to the core polymerase activity, and purely contextual.

**Comment by marcfeuermann** [comment]
Hello, 
I agree with @srengel and @deustp01 . We should keep in mind that we do GO for our users. Many of them would not understand if we remove them. 
Marc.

**Comment by sylvainpoux** [comment]
I agree with @marcfeuermann and @srengel, this will only create confusion in the user community, it is essential to discriminate the activity of the different RNA polymerases.

**Comment by raymond91125** [comment]
We should discuss this as a group. GO:0003899 DNA-directed 5'-3' RNA polymerase activity is defined with https://enzyme.expasy.org/EC/2.7.7.6 that does specifically list pol I, II, and III.
@pgaudet @vanaukenk Should we add this issue to an annotation call? Thanks.

**Comment by pgaudet** [comment]
I am not clear what's the rationale for keeping these. 
- We already have BP and CC, for example [transcription by RNA polymerase III](http://purl.obolibrary.org/obo/GO_0006383) and [RNA polymerase III complex](http://purl.obolibrary.org/obo/GO_0005666) (we have the same for each polymerases). 
- The 5 terms have the exact same definition: 
RNA pol I 
Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase I specific promoter to direct initiation and catalyzes DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.


RNA pol II
Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase II specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.


RNA pol III
Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase III specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.


RNA pol IV
Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase IV specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.

RNA pol V   
Catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template that contains an RNA polymerase V specific promoter to direct initiation and catalyses DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'.

If we can find a way to distinguish them, we can consider re-instating the terms, but as they are defined, these represent gene products. This case is simlar to the caspase case, where we decided that even though the term is used in the literature, the activity is no different from 'cysteine-type peotease' ([see discussion here](https://github.com/geneontology/go-ontology/issues/28468))

**Comment by pgaudet** [comment]
Added synonyms to RNA polymerase activity 
+synonym: "RNA polymerase IV activity" NARROW []
+synonym: "RNA polymerase V activity" NARROW []

**Comment by srengel** [comment]
You send an obsoletion notice 4 days before Christmas, when people are already on break.  
We tell you the obsoletions don’t work for our communities.  
You unilaterally decide that they must be obsoleted based on inadequate definitions as currently written.
Seems like the definitions need to be updated to include that they make different types of RNA and work at different sets of promoters.

**Comment by pgaudet** [comment]
@ValWood you agreed with this is 2022 https://github.com/geneontology/go-ontology/issues/23779#issuecomment-1217536119


Later you wrote 
> We need adequate differentia if they stay. 

I absolutely agree with this. @deustp01 mentions these differences

>The three RNA polymerases involve distinct protein families as catalysts and accessory proteins, recognize distinct elements in genomic DNA to initiate RNA synthesis, and of course mediate the synthesis of structurally and functionally distinct RNAs,

which seems like differences at the CC and BP levels. The catalytic activities are the same. 

We could restore them, but we need to improve  the definitions. Anyone has a suggestion? I am sorry if I missed the proposal.

**Comment by ValWood** [comment]
Personally I think it goes against the rules at the core of GO (not representing gene products), to keep them.

The definitions are identical, and  the differentia will be arbitrary and not derived from the activities, but from other associated genes in the process. This is indicated by the fact that the core catalytic residues are identical.

I don't think it is necessary to have these artificially differentiated functions  for users to understand that they have the same activity, but are involved in different processes when part of different complexes. 
These terms were removed in the past, and then readded, (I ;don't remember the precise history of this). However, when it was discussed that they might go again, I never re-made the more specific annotations. Nobody has ever complained about  this, because these gene products can  be accurately differentiated by their product descriptions e.g.

<img width="633" alt="Image" src="https://github.com/user-attachments/assets/dd78be07-0590-44dc-9645-5695190235c0" />

and it is very easy for users to see the context from the process and component terms:

<img width="497" alt="Image" src="https://github.com/user-attachments/assets/64c73095-a1f2-47ac-9de8-be7a479a1750" />

It doesn't affect enrichments because you really need to enrich over process not function (Functions are small processes).

Finally, the new terms will not map to Rhea, and I prefer to use MF terms that map to Rhea for catalytic activities.

However, if there seems to be quite a strong objection to removing them, so maybe  if there are valid reasons that outweigh the   issues related to annotation consistency and standardization, GO annotation best practice, and ontology best practice and the founding principles of GO then we should reconsider.

**Comment by raymond91125** [comment]
ChEBI does have different RNAs https://www.ebi.ac.uk/chebi/chebiOntology.do?chebiId=CHEBI:33697. Not sure the list is complete for this discussion purpose.
RHEA:21248 does not distinguished between EC 2.7.7.6 DNA-directed RNA polymerase and EC 2.7.7.48 RNA-directed RNA polymerase. RNA(n) + a ribonucleoside 5'-triphosphate = RNA(n+1) + diphosphate. 
Perhaps RHEA can expand on this 'branch' by adding:
mRNA(n) + a ribonucleoside 5'-triphosphate = mRNA(n+1) + diphosphate
tRNA(n) + a ribonucleoside 5'-triphosphate = tRNA(n+1) + diphosphate
... etc.
@amorgat @alexsign Is this something RHEA plan to do?

**Comment by marcfeuermann** [comment]
Exactly, this is what I meant... and there is no reason it is not possible to do it ;-)

**Comment by pgaudet** [comment]
You can also have a look at this document
https://docs.google.com/document/d/1h_NkPLtVQ6uUCsAatlF9h5dW1SeEfaXhE2OmACUA7ZE/edit?tab=t.0

We had proposed in the transcription group to split transcription according to the product made by the polymerase (mRNA, tRNA, etc), however there were limitations to this suggestion, because some products are not transcribed by a single polymerase. 

Just something to keep in mind. We would have the same issue in the MF branch.

**Comment by vanaukenk** [comment]
@deustp01 @pgaudet @marcfeuermann @raymond91125 @ValWood @srengel @colinlog @krchristie @sylvainpoux @thomaspd 

I am working on the agenda for February 4th's annotation call and would like to know if people want to discuss this still-open ticket about MF terms for the various types of RNA polymereases on the call.  

If you feel the issue is resolved, please also let us know.

Thanks; much appreciated.

**Comment by ValWood** [comment]
I think it should be discussed. We have a problem to define if we reinstate.

We absolutely cannot use
mRNA(n) + a ribonucleoside 5'-triphosphate = mRNA(n+1) + diphosphate
tRNA(n) + a ribonucleoside 5'-triphosphate = tRNA(n+1) + diphosphate
etc

because 
pol I transcribes  most, but not all rRNA
poil II  transcribes various mRNA, most, but not all snRNA, other ncRNA
pol III tRNA and some rRNA (5S)

These different contexts are also not a feature of the polymerase activity, but of auxiliary subunits and associated complexes (i.e. process).

Currently, we don't really have a workable solution.

**Comment by deustp01** [comment]
@vanaukenk My comment above is the totality of what I know about the subject so while my best guess is that there are still issues to resolve here, I don't know how to do it.

**Comment by sjm41** [comment]
For what it's worth, EC has a single entry covering RNAP I/II/III:

https://enzyme.expasy.org/EC/2.7.7.6

**Comment by ValWood** [comment]
Rhea has only  5'-3' RNA polymerase activity https://www.rhea-db.org/rhea/21248

**Comment by raymond91125** [comment]
I'd like to close ticket since there is no further action to be done. Should further discussions are needed, please reopen or start a new ticket. Thanks, y'all.
